src/nextjs/nextjs-wizard.ts

```diff
@@ -116,12 +118,46 @@ export async function runNextjsWizardWithTelemetry(
       forceInstall,
     });
 
+  let selectedProject: SentryProjectData;
+  let authToken: string;
+  let selfHosted: boolean;
+  let sentryUrl: string;
+  let spotlight: boolean;
+  let dsn: string;
+
+  if (projectData.spotlight) {
+    // Spotlight mode: use empty DSN and skip auth
+    spotlight = true;
+    dsn = '';
+    selfHosted = false;
+    sentryUrl = '';
+    authToken = '';
+    // Create a minimal project structure for type compatibility
+    selectedProject = {
+      id: '',
+      slug: '',
+      organization: { id: '', slug: '', name: '' },
+      keys: [{ dsn: { public: '' } }],
+    };
```

Comment:
l/feel free to ignore: Probably fine to extract if we indeed need the same data in all wizards supporting spotlight. Feel free to disregard for now and adjust as necessary in future PRs.